command/jenkinsjson.go

```diff
@@ -236,20 +247,29 @@ func (c *JenkinsJson) processFile(filePath string, file *os.File) subcommands.Ex
 		return subcommands.ExitFailure
 	}
 
-	// convert the pipeline yaml from the jenkinsjson
-	// format to the harness yaml format.
-	converter := jenkinsjson.New(
-		jenkinsjson.WithDockerhub(c.dockerConn),
-		jenkinsjson.WithKubernetes(c.kubeName, c.kubeConn),
-	)
+	// create converter with options
+	options := []jenkinsjson.Option{}
+
+	// add infrastructure options if specified
+	if c.infrastructure != "" {
+		options = append(options, jenkinsjson.WithInfrastructure(c.infrastructure))
+	}
+	if c.os != "" {
+		options = append(options, jenkinsjson.WithOS(c.os))
+	}
+	if c.arch != "" {
+		options = append(options, jenkinsjson.WithArch(c.arch))
+	}
+
+	// convert the pipeline yaml from the jenkinsjson format to the harness yaml format
+	converter := jenkinsjson.New(options...)
```

Comment:
Looks like we are losing the `WithDockerhub` and `WithKubernetes` parameters, was this intentional?

Reply:
@sean-harness Thanks for catching. Seems like formatting time issue. I'll fix it in a subsequent patch.